api/src/main/java/org/openmrs/module/providermanagement/ProviderManagementUtils.java

```diff
@@ -99,6 +104,66 @@ public static Date clearTimeComponent(Date date) {
 
         return cal.getTime();
     }
+
+    public static List<ProviderPersonRelationship> getSupervisors(Provider provider)
```

Comment:
Actually, it looks like these are methods you originally wrote in coreapps but migrated here?  I think that makes sense, but in the documentation let's note that coreapps uses these methods for reference?